tests/microos/toolbox.pm

```diff
@@ -71,6 +71,9 @@ sub run {
         # $image = suse/sle-15-sp3/update/products/microos51/update/cr/images/suse/sle-micro/5.1/toolbox:latest
         (my $registry = $toolbox_image_to_test) =~ s/\/.*//;
         (my $image = $toolbox_image_to_test) =~ s/^[^\/]*\///;
+        if ($registry eq "registry.opensuse.org") {
```

Comment:
I find this a bit of a dirty hack. I think MicroOS is special and there I would skip the whole handling of the `CONTAINER_IMAGE_TO_TEST` setting instead of replacing parts of the passed variable.

Reply:
The main problem is that `CONTAINER_IMAGE_TO_TEST` is used for different test modules. In https://openqa.opensuse.org/tests/4932982 it is rightfully set to `registry.opensuse.org/opensuse/factory/totest/containers/opensuse/tumbleweed` for the podman tests.

The conflict happens then when the `toolbox` test run uses the same variable, but would require the toolbox image.

***

I suggest we avoid this conflict by not using this variable for MicroOS at all but hard-code the toolbox image location there in the test module.

Reply:
Alternatively one would need to separate the `podman` and the `toolbox` tests into two test runs.

Reply:
Fixed by hard-coding it on TW.